### PR TITLE
Update transaction creation according to Lisk SDK v6 schema - Closes #4400

### LINKED ISSUE #4400: Update transaction creation according to Lisk SDK v6 schema
### Description

Adopt transaction creation according to Lisk SDK v6 schema.

- Update `@liskhq/lisk-client` to latest v6 alpha version
- Update transaction creation, validation, sign and broadcast according to Lisk SDK v6 schema

### Acceptance Criteria

- Should be able to create all types of transaction with v6 transaction schema
- Should be able to submit all types of transaction with v6 transaction schema

### Additional information
- Refer transaction schema https://github.com/LiskHQ/lisk-sdk/blob/development/elements/lisk-transactions/src/schema.ts

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #4400

Note: This PR is blocked by https://github.com/LiskHQ/lisk-sdk/pull/7415, once the SDK PR is merged and we have next alpha, this PR can be resolved.

### How was it solved?

- [x] Update transaction creation according to Lisk SDK v6 schema
- [x] Integrate endpoints to fetch `baseFees`
- [x]  Replace `networkIdentifier` with `chainID`
- [x] Refactor static transaction schema and transaction usage with endpoint to fetch transaction information and schema `networkStatus (nodeInfo and transaction Schema)`
### How was it tested?

- Updated all tests

Test by modifying any component

- Wallet Connect Example
  - Wallet connect will send sign transaction request with encoded transaction has bytes in `hex` format
  - Steps to sign or view summary
   - useCreateTransaction hook
   - sign (returns bytes)
   - for summary view call `.toJSON`
```
  const encodedTransaction = '0a05746f6b656e12087472616e73666572180020a0fb0b2a20fd061b9146691f3c56504be051175d5b76d1b1d0179c5c4370e18534c588212232470a0800000002000000011080c2d72f1a1400b1182b317a82e4b9c4a54119ced29f19b496de22207765206861766520736f6d6520696e666f726d6174696f6e2069732068657265';
  const walletConnectTrx = useCreateTransaction({
    encodedTransaction,
  });

  useEffect(() => {
    if (!walletConnectTrx.isLoading) {
      walletConnectTrx.sign(
        'YOUR PRIVATE KEY',
      ).then(() => {
        console.log('.'.repeat(10), {
          signedTrx: walletConnectTrx.transaction,
          trxJSON: walletConnectTrx.toJSON(),
          signedBytesToReturn: walletConnectTrx.encode().toString('hex'),
        });
      });
    }
  }, [walletConnectTrx.isLoading]);
```
- Lisk Desktop Example
```
  const tokenTransfer = useCreateTransaction({
    module: 'token',
    command: 'transfer',
  });
  useEffect(() => {
    if (!tokenTransfer.isLoading) {
      const tokenTransferParams = {
        tokenID: '0000000200000001',
        amount: '100000000', // amount in beddows
        recipientAddress: '00b1182b317a82e4b9c4a54119ced29f19b496de',
        accountInitializationFee: '5000000',
        data: 'we have some information is here',
      };

      tokenTransfer.update({ params: tokenTransferParams });
      console.log(':'.repeat(10), {
        encodedTrx: tokenTransfer.encode().toString('hex'),
      });
      tokenTransfer.sign(
        'YOUR PRIVATE KEY',
      ).then(() => {
        console.log('.'.repeat(10), {
          signedTrx: tokenTransfer.transaction,
          trxJSON: tokenTransfer.toJSON(),
          encodedTrx: tokenTransfer.encode().toString('hex'),
        });
      });
    }
  }, [tokenTransfer.isLoading]);
 
 ```

### CODE REVIEW

**Comment by ManuGowda** [comment]
This PR is blocked by https://github.com/LiskHQ/lisk-sdk/pull/7415, once the SDK PR is merged and we have next alpha, this PR can be resolved.

File: .nvmrc
Comment (clemente-xyz) [inline .nvmrc]: why was this made?

File: src/modules/common/components/filterDropdownButton/index.js
Comment (clemente-xyz) [inline src/modules/common/components/filterDropdownButton/index.js]: and `'5:3'`?

File: src/modules/common/components/filterDropdownButton/selectFilter.js
Comment (clemente-xyz) [inline src/modules/common/components/filterDropdownButton/selectFilter.js:13]: not required to change, but `Object.entries` makes the trick here ;)

File: src/modules/dpos/validator/utils/getMaxAmount.test.js
Comment (clemente-xyz) [inline src/modules/dpos/validator/utils/getMaxAmount.test.js:37]: same. What happened here?

File: src/modules/legacy/components/Summary/Summary.js
Comment (clemente-xyz) [inline src/modules/legacy/components/Summary/Summary.js]: maybe `@transaction/` instead of `src/modules/` ?

File: src/modules/network/utils/api/index.test.js
Comment (clemente-xyz) [inline src/modules/network/utils/api/index.test.js]: same

File: src/modules/token/fungible/components/SendForm/SendForm.js
Comment (clemente-xyz) [inline src/modules/token/fungible/components/SendForm/SendForm.js:5]: maybe @transaction/ instead of src/modules/ ?

File: src/modules/transaction/api/index.js
Comment (clemente-xyz) [inline src/modules/transaction/api/index.js:90]: is missing to update the IDs to names on the comment :)

File: src/modules/transaction/api/index.js
Comment (clemente-xyz) [inline src/modules/transaction/api/index.js:14]: why dont we use axios for transactions API calls?

File: src/modules/transaction/hooks/useCreateTransaction.js
Comment (clemente-xyz) [inline src/modules/transaction/hooks/useCreateTransaction.js]: some deps are missing (e.g. `transaction`). Can you add them guys?

File: src/modules/transaction/hooks/useCreateTransaction.js
Comment (clemente-xyz) [inline src/modules/transaction/hooks/useCreateTransaction.js:27]: i personally prefer to check on `data` rather than on `loading` on this cases. Might be that the API calls finishes loading and fall in `error`. A transition to `isLoading: false` doesn't guarantees that `data` will be defined. I personally prefer to use `isLoading` booleans for UI purposes, rather that business logic.

File: src/modules/transaction/utils/encoding.js
Comment (clemente-xyz) [inline src/modules/transaction/utils/encoding.js:19]: why dont we implement the `useSchemaQuery` hook instead? (`GET /commands/parameters/schemas`)

File: src/modules/transaction/utils/moduleCommand.js
Comment (clemente-xyz) [inline src/modules/transaction/utils/moduleCommand.js:6]: this `t` mapping could be much more straight forward using the `i18next.t()` singleton :)

Comment (clemente-xyz) [reply src/modules/transaction/utils/moduleCommand.js:6]: with this, we can just do:

```js
// ...
import i18next from "i18next"

const getModuleCommandSenderLabel = ({
  [MODULE_COMMANDS_NAME_MAP.transfer]: i18next.t('Sender'),
  // ...
})
// ...

```

**Review by clemente-xyz — CHANGES_REQUESTED** [review]
Great! Left just a few comments :)

File: .nvmrc
Comment (ManuGowda) [reply .nvmrc]: to fix the warnings, i think if you run the npm i you will see the problem with 16.14.0

File: src/modules/transaction/utils/moduleCommand.js
Comment (ManuGowda) [reply src/modules/transaction/utils/moduleCommand.js:6]: For now we can leave it as it is, my feedback on this implementation would be when we refactor the multistep component.

Each Command should have props `name, title` so we don't need this logic. Then essentially Each command and params will have its name and its title.

File: src/modules/transaction/utils/encoding.js
Comment (ManuGowda) [reply src/modules/transaction/utils/encoding.js:19]: I think once SDK exposes the schema via elements we can use the schema from elements.

File: src/modules/transaction/hooks/useCreateTransaction.js
Comment (ManuGowda) [reply src/modules/transaction/hooks/useCreateTransaction.js:27]: sounds good, @soroushm ?

File: src/modules/common/components/filterDropdownButton/selectFilter.js
Comment (ManuGowda) [reply src/modules/common/components/filterDropdownButton/selectFilter.js:13]: As i mentioned below this logic should be refactored, so i am not going to change anything for now, added a TODO.

**Comment by ManuGowda** [comment]
@ClementeSerrano I have addressed few comments, I haven't addressed few other comments, there are several refactoring needs to be done which i haven't focused in this PR. The main logic change would be with regards to `transactions.js` class implementation 🙏🏼

File: src/modules/common/components/filterDropdownButton/selectFilter.js
Comment (clemente-xyz) [reply src/modules/common/components/filterDropdownButton/selectFilter.js:13]: 🆗

File: src/modules/transaction/utils/encoding.js
Comment (clemente-xyz) [reply src/modules/transaction/utils/encoding.js:19]: ok

File: src/modules/transaction/utils/moduleCommand.js
Comment (clemente-xyz) [reply src/modules/transaction/utils/moduleCommand.js:6]: ok

File: src/modules/transaction/api/index.js
Comment (clemente-xyz) [reply src/modules/transaction/api/index.js:14]: ok, we will consider to refactor to Axios in a future.

**Review by clemente-xyz — DISMISSED** [review]
Perfect!

**Review by clemente-xyz — APPROVED** [review]

**Comment by ManuGowda** [comment]
@reyraa @soroushm I will go ahead and merge the PR, do not want to keep it open. 

In summary:
- This PR includes changes to transaction schema and updates to utility
  - The transaction creation flow is fully not working, this requires a decision on approach that we are discussing internally. After deciding on new approach our transaction creation and signing flow should work 
- The new transaction class and `useCreateTransaction` hook can be used or removed based on direction

**Pull request merged**
